CatalogSrvc/azure-pipelines.yml

```diff
@@ -0,0 +1,37 @@
+resources:
+- repo: self
+queue:
+  name: Hosted Ubuntu 1604
+  demands: java
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - CatalogSrvc/*
+
+steps:
+- task: Gradle@2
+  displayName: 'gradlew build'
+  inputs:
+    gradleWrapperFile: CatalogSrvc/gradlew
+    workingDirectory: CatalogSrvc
+
+- task: Docker@1
+  displayName: 'Build an image'
+  inputs:
+    azureSubscriptionEndpoint: $(SERVICE_CONNECTION)
+    azureContainerRegistry: pumrpmicroacr.azurecr.io

```

Comment:
Let's put this ACR value to a variable throughout and then the rest looks good!

Reply:
updated these to $(ACR_SERVER) and set that variable in the azure devops pipelines - in the same location as the $(SERVICE_CONNECTION) variable